wpt/tests/serialization/serialization.py

```diff
@@ -0,0 +1,206 @@
+                               recursive_compare)
+
+
+@pytest.mark.asyncio
+async def test_serialization_null(send_blocking_command, context_id,
+      recursive_compare):
+    await assert_serialization("null", {
+        "type": "null"}, send_blocking_command, context_id, recursive_compare)
+
+
+@pytest.mark.asyncio
+async def test_serialization_string(send_blocking_command, context_id,
+      recursive_compare):
+    await assert_serialization("'someStr'", {
+        "type": "string",
+        "value": "someStr"}, send_blocking_command, context_id,
+                               recursive_compare)
+
+
+@pytest.mark.asyncio
+async def test_serialization_number(send_blocking_command, context_id,
+      recursive_compare):
+    await assert_serialization("123", {
+        "type": "number",
+        "value": 123}, send_blocking_command, context_id, recursive_compare)
+    await assert_serialization("0.56", {
+        "type": "number",
+        "value": 0.56}, send_blocking_command, context_id, recursive_compare)
+
+
+@pytest.mark.asyncio
+async def test_serialization_specialNumber(send_blocking_command, context_id,
+      recursive_compare):
+    await assert_serialization("+Infinity", {
+        "type": "number",
+        "value": "+Infinity"}, send_blocking_command, context_id,
+                               recursive_compare)
+    await assert_serialization("-Infinity", {
+        "type": "number",
+        "value": "-Infinity"}, send_blocking_command, context_id,
+                               recursive_compare)
+    await assert_serialization("-0", {
+        "type": "number",
+        "value": "-0"}, send_blocking_command, context_id, recursive_compare)
+    await assert_serialization("NaN", {
+        "type": "number",
+        "value": "NaN"}, send_blocking_command, context_id, recursive_compare)
+
+
+@pytest.mark.asyncio
+async def test_serialization_bool(send_blocking_command, context_id,
+      recursive_compare):
+    await assert_serialization("true", {
+        "type": "boolean",
+        "value": True}, send_blocking_command, context_id, recursive_compare)
+    await assert_serialization("false", {
+        "type": "boolean",
+        "value": False}, send_blocking_command, context_id, recursive_compare)
+
+
+@pytest.mark.asyncio
+async def test_serialization_function(send_blocking_command, context_id,
+      recursive_compare):
+    await assert_serialization("function(){}", {
+        "type": "function",
+        "objectId": "__any_value__"
+    }, send_blocking_command, context_id, recursive_compare)
+
+
+@pytest.mark.asyncio
+async def test_serialization_object(send_blocking_command, context_id,
+      recursive_compare):
+    await assert_serialization("{'foo': {'bar': 'baz'}, 'qux': 'quux'}", {
+        "type": "object",
+        "objectId": "__any_value__",
+        "value": [[
+            "foo", {
+                "type": "object",
+                "objectId": "__any_value__"}], [
```

Comment:
The value for `{'bar': 'baz'}` is omitted here. Should it be serialized?

Reply:
The default serialization max_depth is 1, so it shouldn't be there.